modules/wallet/persist.go

```diff
@@ -129,3 +137,19 @@ func (w *Wallet) load() error {
 
 	return nil
 }
+
+// MergeWallet merges another wallet with the already-loaded wallet, creating a
+// new wallet that contains all of the addresses from each. This is useful for
+// loading backups.
+func (w *Wallet) MergeWallet(filepath string) error {
+	var savedKeys []savedKey
+	err := encoding.ReadFile(filepath, &savedKeys)
+	if err != nil {
+		return err
+	}
+	err = w.loadKeys(savedKeys)
+	if err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
initially it wasn't, but I think I changed load to work like Merge.

`load` also grabs outputs, but I can deduplicate a lot of this.